build.sbt

```diff
@@ -5,6 +5,7 @@ val commonSettings: Seq[Setting[_]] = Seq(
   version := "0.1.1-SNAPSHOT",
   organization := "org.scala-js",
   scalaVersion := "2.11.8",
+  crossScalaVersions := Seq("2.10.4", "2.11.8", "2.12.0"),
```

Comment:
`crossScalaVersions` should be `in ThisBuild` at the top-level of the .sbt file, like this:
```scala
import sbt.Keys._
import org.scalajs.sbtplugin.cross.CrossProject

crossScalaVersions in ThisBuild := Seq("2.10.4", "2.11.8", "2.12.0")

val commonSettings: Seq[Setting[_]] = Seq(
...
```